src/app/store/service/types/enum.ts

```diff
@@ -19,6 +20,7 @@ export enum ServiceName {
   SYSLOG_RACK = "syslog_rack",
   SYSLOG_REGION = "syslog_region",
   TFTP = "tftp",
+  TEMPORAL_SERVER = "temporal",
```

Comment:
Nitpicking, but this will make it consistent with the rest of enum members.

```suggestion
  TEMPORAL = "temporal",
```